packages/ui-react/src/components/LanguageMenu/LanguageMenu.tsx

```diff
@@ -60,10 +61,12 @@ const LanguageMenu = ({ onClick, className, languages, currentLanguage, language
           <ul className={styles.menuItems}>
             {languages.map(({ code, displayName }) => {
               const menuItemClassname = classNames(styles.menuItem, { [styles.menuItemActive]: currentLanguage?.code === code });
-
+              if (currentLanguage?.code === code) {
+                ariaAttributes['aria-current'] = 'true';
+              }
               return (
                 <li key={code} className={menuItemClassname} onClick={(event) => handleLanguageSelect(event, code)}>
-                  <Link onFocus={openLanguageMenu} onBlur={closeLanguageMenu} href="#">
+                  <Link onFocus={openLanguageMenu} onBlur={closeLanguageMenu} href="#" {...ariaAttributes}>
```

Comment:
With the change from @langemike ^

The `ariaAttributes` is initialized outside the loop. This means once a property is set, it isn't removed anymore resulting in all items receiving the same aria attributes.

```suggestion
                  <Link onFocus={openLanguageMenu} onBlur={closeLanguageMenu} href="#" aria-current={isActive ? 'true' : undefined}>
```